benchmark/bench_dom_api.cpp

```diff
@@ -73,6 +73,79 @@ BENCHMARK(serialize_twitter)->Repetitions(10)->ComputeStatistics("max", [](const
     return *(std::max_element(std::begin(v), std::end(v)));
   })->DisplayAggregatesOnly(true);
 
+
+static void serialize_twitter_to_string(State& state) {
+  dom::parser parser;
+  padded_string docdata;
+  auto error = padded_string::load(TWITTER_JSON).get(docdata);
+  if(error) {
+      cerr << "could not parse twitter.json" << error << endl;
+      return;
+  }
+  // we do not want mem. alloc. in the loop.
+  if((error = parser.allocate(docdata.size()))) {
+      cout << error << endl;
+      return;
+  }
+  dom::element doc;
+  if ((error = parser.parse(docdata).get(doc))) {
+    cerr << "could not parse twitter.json" << error << endl;
+    return;
+  }
+  size_t bytes = 0;
+  for (SIMDJSON_UNUSED auto _ : state) {
+    auto serial = simdjson::to_string(doc);
```

Comment:
It's absolutely optional, but it might be worth creating a jsonexamples/minified/twitter.json and comparing the results to make sure we're doing it right. (Maybe just as a test.)

Reply:
I have added tests, directly as part of the benchmarks ( at the end) and also as a supplementary tests.